### ISSUE TITLE
Snake case for schema

### ISSUE THREAD

**Issue by apkrymov** [proudmonkey/AutoWrapper#74]
Are there any plans for implementation custom property naming policy for default responses, e.t.c snake_case?

**Comment by proudmonkey** [comment]
Yes. The plan is also migrate or atleast support `System.Text.Json`. I plan to release a new version once .NET Core 5 is out.
